legate/core/operation.py

```diff
@@ -279,7 +277,7 @@ def can_raise_exception(self) -> bool:
         return len(self._exn_types) > 0
 
     def capture_traceback(self) -> None:
-        self._tb = capture_traceback()
+        self._tb = capture_traceback_repr()
```

Comment:
`self._tb_repr`

Reply:
not sure why mypy isn't flagging this tbh

Reply:
maybe we need to explicitly enable such a check?